fmriprep/workflows/bold/base.py

```diff
@@ -424,6 +425,9 @@ def init_func_preproc_wf(bold_file, ignore, freesurfer,
                                              omp_nthreads=omp_nthreads,
                                              use_compression=False)
 
+    # get non steady-state volumes
+    non_steady_state = pe.Node(nac.NonSteadyStateDetector(), name='non_steady_state')
```

Comment:
You should already be able to get this from `bold_reference_wf.outputnode.skip_vols`.

Reply:
nice, thanks!